### PR TITLE
update/version-in-ui

### PR DESCRIPTION
**Pull request by markmead**
Shows the current GetCandy version in the UI.

If it's a semver version it defaults to `2.0-beta`, do you think this should be set in the config or nah?

### CODE REVIEW

File: packages/admin/src/Views/Components/GetCandy/Version.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+use Illuminate\Support\Str;
+use Illuminate\View\Component;
+
+class Version extends Component
+{
+    public $installedVersion;
+
+    public function __construct()
+    {
+        $installedVersion = InstalledVersions::getPrettyVersion('getcandy/getcandy');
```
Comment (alecritson) [inline packages/admin/src/Views/Components/GetCandy/Version.php:15]: 🤯

**Comment by alecritson** [comment]
Just pulled down the branch and the menu toggle is now appearing at the top 
image

**Comment by markmead** [comment]
Odd, have you published the CSS?
Screenshot 2022-09-05 at 10 13 39

**Comment by alecritson** [comment]
@markmead Just republished the assets and still the same, does the css need to be recompiled on the branch?

**Comment by markmead** [comment]
Yeah, that's what I did @alecritson

**Pull request closed without merging**